### DISCUSSION TITLE
UXEL Fooocus - Just have a look

### DISCUSSION THREAD

**Discussion by nick-arch** [lllyasviel/Fooocus#2841] [Q&A] [answered]
![Screenshot_20240501_143106_Chrome](https://github.com/lllyasviel/Fooocus/assets/57282548/db4cc343-c4cc-4de5-a3a6-43b3af4c3bd3)

Hey Buddy, I wanted to give you a heads up—I've simplified the model downloading process in the Fooocus sdxl 1.0 project on Google Colab. It's now much more user-friendly. Feel free to check it out whenever you get a chance!

![Screenshot_20240501_143127_Chrome](https://github.com/lllyasviel/Fooocus/assets/57282548/0263dbbc-08b2-48fc-a1d1-ba7fd625b61d)

**Comment by CantainPike** [comment]
I can not see the Fooocus here, all beautiful elegante UIX design gone, probably only some parts of the engine works. Car without bodywork

**Reply by ILikeToasters** [reply]
I think that is just showing the Colab part and not Fooocus running.

**Reply by wisdombox44** [reply]
You are right :)

**Comment by nick-arch** [comment, marked as answer]
https://colab.research.google.com/drive/1ch7vpHFFdkc4W48xDO3bgLMzhuGMgd8-?usp=sharing

I forgot, here is Link

**Reply by tommy3210** [reply]
I love it, it beautiful
